src/App.jsx

```diff
@@ -14,33 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABETICALLY_ID = 'alphabetically';
+const SORT_LENGTH_ID = 'by_length';
+const SORT_REVERSE_ID = 'reverse';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReverse, setIsReverse] = useState(false);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const handleSortClick = newSortField => {
+    if (newSortField === SORT_REVERSE_ID) {
+      setIsReverse(!isReverse);
+    } else {
+      setSortField(newSortField);
+    }
+  };
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  const getPreparedGoods = (goods, currentField) => {
```

Comment:
You can move it outside the component, then import and use it as a helper/util function (for feature)